src/main/java/edu/itu/cavabunga/caldav/exception/CalendarRestException.java

```diff
@@ -0,0 +1,4 @@
+package edu.itu.cavabunga.caldav.exception;
+
+public class CalendarRestException {
```

Comment:
Is there any exception base class on Spring boot? Maybe, we can extend this class